runtime/modules/isp_vcu118.py

```diff
@@ -279,7 +282,7 @@ def runStock(exe_path, ap, openocd_log_file, gdb_log_file,
         return isp_utils.retVals.FAILURE
 
     logger.debug("Spawning gdb")
-    gdb = threading.Thread(target=gdb_thread, args=(exe_path, gdb_log_file))
+    gdb = threading.Thread(target=gdb_thread, args=(exe_path, gdb_log_file, "rv32"))
```

Comment:
Don't hard code "rv32". Use the `arch` argument instead

Reply:
Fixed